server/index.js

```diff
@@ -10,10 +10,12 @@ var corsHeaders = require('hapi-cors-headers')
 var hoodieServer = require('@hoodie/server').register
 var log = require('npmlog')
 var PouchDB = require('pouchdb-core')
+var cloneDeep = require('lodash').cloneDeep
```

Comment:
> No blocker tho

I've adapted from here: https://github.com/hoodiehq/hoodie/blob/7a989154c766419a1f73c29bc5a2860e94c470aa/cli/options.js#L3

But yeah, found out about the `CODING_STYLE.md` now, will create another PR that fixes this inside of `server/index.js`.